src/commands/database/database.ts

```diff
@@ -0,0 +1,276 @@
+import fs from 'fs'
+import path from 'path'
+import { OptionValues } from 'commander'
+import BaseCommand from '../base-command.js'
+
+import openBrowser from '../../utils/open-browser.js'
+import { carefullyWriteFile, getExtension, getExtensionInstallations, installExtension } from './utils.js'
+import { getToken } from '../../utils/command-helpers.js'
+import inquirer from 'inquirer'
+import { NetlifyAPI } from 'netlify'
+import { spawn } from 'child_process'
+
+const NETLIFY_DATABASE_EXTENSION_SLUG = '-94w9m6w-netlify-database-extension'
+
+const init = async (_options: OptionValues, command: BaseCommand) => {
+  process.env.UNSTABLE_NETLIFY_DATABASE_EXTENSION_HOST_SITE_URL = 'http://localhost:8989'
+
+  if (!command.siteId) {
+    console.error(`The project must be linked with netlify link before initializing a database.`)
+    return
+  }
+
+  const initialOpts = command.opts()
+
+  const answers = await inquirer.prompt(
+    [
+      {
+        type: 'confirm',
+        name: 'drizzle',
+        message: 'Use Drizzle?',
+      },
+    ].filter((q) => !initialOpts[q.name]),
+  )
+
+  if (!initialOpts.drizzle) {
+    command.setOptionValue('drizzle', answers.drizzle)
+  }
```

Comment:
wouldn't we only want to prompt if they don't provide it as an optional
```suggestion
    if (!initialOpts.drizzle) {
  const answers = await inquirer.prompt(
    [
      {
        type: 'confirm',
        name: 'drizzle',
        message: 'Use Drizzle?',
      },
    ].filter((q) => !initialOpts[q.name]),
  )


    command.setOptionValue('drizzle', answers.drizzle)
  }
```

Reply:
Definitely! 

I updated `--drizzle` to be a negatable boolean option as well `--no-drizzle` and am now not prompting here unless `initialOpts.drizzle !== true && initialOpts.drizzle !== false`. 

If both `--drizzle` and `--no-drizzle` are not passed then `initialOpts.drizzle` will be `undefined`.